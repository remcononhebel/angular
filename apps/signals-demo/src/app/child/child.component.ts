import { Component } from '@angular/core';

@Component({
    selector: 'signals-demo-child',
    templateUrl: './child.component.html',
    styleUrls: ['./child.component.scss'],
    standalone: true
})
export class ChildComponent {

}
