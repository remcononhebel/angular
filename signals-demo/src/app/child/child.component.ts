import { Component } from '@angular/core';

@Component({
    selector: 'app-child',
    templateUrl: './child.component.html',
    styleUrls: ['./child.component.scss'],
    standalone: true
})
export class ChildComponent {

}
