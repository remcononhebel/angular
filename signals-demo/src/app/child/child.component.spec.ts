import { Spectator, createComponentFactory } from '@ngneat/spectator';
import { ChildComponent } from './child.component';

describe('ChildComponent', () => {
  let spectator: Spectator<ChildComponent>;
  const createComponent = createComponentFactory(ChildComponent);

  beforeEach(() => spectator = createComponent());

  it('should create', () => {
    expect(spectator.component).toBeTruthy();
  });

  it('should render labels and inputs', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
