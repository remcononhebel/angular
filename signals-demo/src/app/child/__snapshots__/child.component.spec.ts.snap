// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChildComponent should render labels and inputs 1`] = `
<app-child>
  <p>
    child works!
  </p>
</app-child>
`;
